### PR TITLE
leaflet.esri.related

### PR DESCRIPTION
**Pull request by jgravois**
i made sure to case this additional esri-leaflet plugin as base**Path**.  thanks y'all!

### CODE REVIEW

File: files/leaflet.esri.related/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "john gravois"
+github = "https://github.com/jgravois/esri-leaflet-related.git"
+homepage = "https://github.com/jgravois/"
+description = "Esri Leaflet Related Records is a small API helper to assist querying related tables published in ArcGIS Server or ArcGIS Online"
+mainfile = "esri-leaflet-related.js"
```
Comment (jimaek) [inline files/leaflet.esri.related/info.ini]: Use the min file here

File: files/leaflet.esri.related/update.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "packageManager": "github",
+  "name": "leaflet.esri.related",
+  "repo": "jgravois/esri-leaflet-related",
+  "files": {
+    "basePath": "dist/",
+    "include": ["esri-leaflet-related.js", "esri-leaflet-related.js.map"],
```
Comment (jimaek) [inline files/leaflet.esri.related/update.json]: Include only the min file and the related map file https://github.com/jgravois/esri-leaflet-related/tree/v0.0.1-beta.2/dist

**Comment by jgravois** [comment]
arghh.  thanks for catching that!

i'm going to update my build process to name the minified output 'esri-leaflet-related.js' and the other 'esri-leaflet-related-src.js' instead (to match the convention of the other plugins) and let you know when you can merge the existing PR.

**Comment by jgravois** [comment]
okay, should be good now.
jgravois/esri-leaflet-related/b300a9920562585de4eab6f9d360aa22e8b65b90

**Comment by jimaek** [comment]
I think you need to tag a new version for the changes to apply

**Comment by jgravois** [comment]
thanks for your patience with me.  i thought i was supposed to wait to publish until after you merged this PR.

i'm all good now.
https://github.com/jgravois/esri-leaflet-related/tree/v1.0.2/dist

**Pull request merged**